bindings/wallet-jni/java/WalletTest.java

```diff
@@ -142,8 +137,20 @@ public void importKeys() throws IOException {
 
     @Test
     public void convertWallet() throws IOException {
-        final long walletPtr = Wallet.recover(
-                "neck bulb teach illegal soul cry monitor claw amount boring provide village rival draft stone");
+        final byte[] accountKey = { -56, 101, -106, -62, -47, 32, -120, -123, -37, 31, -29, 101, -124, 6, -86, 15, 124,
```

Comment:
No easy way to load these from test vector files in Java?

Reply:
Well, reading them is not the problem (we already read the block0 from file after all). The thing is they are saved as bech32 and there is no easy way of decoding that.

Reply:
We could store them as hex or binary files though.

Reply:
Also, extracting them to some functions/methods would be nice, now that I think about it.

Reply:
Hex would be simplest for every language, I suppose.